README.md

```diff
@@ -41,12 +41,14 @@ Usage
 Enable and start the processor and odr-padenc.
 
 ```bash
-systemctl enable dabplus-on-air-processing
-systemctl start dabplus-on-air-processing
+systemctl enable dabplus-odr-padenc@dabplus-on-air-processing
+systemctl enable liquidsoap@dabplus-on-air-processing
+
+systemctl start dabplus-odr-padenc@dabplus-on-air-processing
+systemctl start liquidsoap@dabplus-on-air-processing
 
-systemctl status dabplus-on-air-processing
-systemctl status liquidsoap@dabplus-on-air-processing
 systemctl status dabplus-odr-padenc@dabplus-on-air-processing
+systemctl status liquidsoap@dabplus-on-air-processing
 ```
 
 You may put slides and DL text files into `/var/tmp/mot/dabplus-on-air-processing/{slides,texts}`.
```

Comment:
The path was streamlined to `/var/tmp/odr/padenc/dapplus-on-air-processing/{slides,texts}` see [running-odr-padenc-through-systemd](https://github.com/radiorabe/centos-rpm-odr-padenc#running-odr-padenc-through-systemd) and [197463898](https://github.com/radiorabe/dabplus-on-air-processing/issues/2#issue-197463898)

The path needs also to be changed on the `odr-audioenc` command within [dabplus-on-air-processing.conf](https://github.com/radiorabe/dabplus-on-air-processing/blob/master/dabplus-on-air-processing.conf#L35)